runtime/onert/api/include/nnfw_experimental.h

```diff
@@ -382,7 +382,7 @@ NNFW_STATUS nnfw_set_quantization_type(nnfw_session *session, NNFW_QUANTIZE_TYPE
  * @param[in] path    Quantized model path
  * @return    @c NNFW_STATUS_NO_ERROR if successful, otherwise return @c NNFW_STATUS_ERROR
  */
-NNFW_STATUS nnfw_set_quantized_model_path(nnfw_session *session, const char *path);
+NNFW_STATUS nnfw_quantize_set_qmodel_path(nnfw_session *session, const char *path);
```

Comment:
I am thinking of `working directory`. Because there are several files (e.g. minmax recording, ... to be saved. `quantized model` is not the only one. Also, we need to generate the quantized model under some rule, not arbitrary name by user.